pkg/ui/src/views/app/components/layoutSidebar/index.tsx

```diff
@@ -128,6 +137,13 @@ export default class Sidebar extends React.Component {
           <IconLink to="/jobs" icon={jobsIcon} title="Jobs" />
         </ul>
         <ul className="navigation-bar__list navigation-bar__list--bottom">
+          <li>
+            <Link to="/debug" className={classNames("debug-pages-link", { active: this.context.router.isActive("/debug") })}>
```

Comment:
please use `IconLink`

Reply:
Tried to use `IconLink` — it pulls in a lot of CSS which I was unable to finagle to get the spacing I wanted. Maybe will be more successful today.